lua/ror/test.lua

```diff
@@ -2,6 +2,48 @@ local config = require("ror.config").values.test
 
 local M = {}
 
+function M.non_empty_last_line(bufnr)
+  local lines = vim.fn.getbufline(bufnr, 1, '$')
+
+  local line = ""
+
+  for i = #lines, 1, -1 do
+    if lines[i] ~= "" then
+      line = lines[i]
+
+      break
+    end
+  end
+
+  return line
+end
+
+function M.verify_debugger()
```

Comment:
I am thinking we might not want to expose this function
```suggestion
local function verify_debugger()
```